cmd/ctr/utils.go

```diff
@@ -79,3 +85,68 @@ func waitContainer(events execution.ContainerService_EventsClient, id string, pi
 		}
 	}
 }
+
+func forwardAllSignals(containers execution.ContainerServiceClient, id string) chan os.Signal {
+	sigc := make(chan os.Signal, 128)
+	catchAll(sigc)
+	go func() {
+	Signals:
+		for s := range sigc {
+			var sig syscall.Signal
+			for sigStr, sigN := range signalMap {
+				if sigN == s {
+					var err error
+					sig, err = parseSignal(sigStr)
+					if err != nil {
+						logrus.WithError(err).Error("parse signal error")
+						continue Signals
+					}
+					break
+				}
+			}
+
+			logrus.Debug("Forwarding signal:", sig)
+			killRequest := &execution.KillRequest{
+				ID:     id,
+				Signal: uint32(sig),
+				All:    true,
```

Comment:
I think this should be false.  Only signal the initial process that awe are interacting with